themes/OpenFisca-2022/layouts/partials/footer.html

```diff
@@ -0,0 +1,26 @@
+<footer class="footer">
+	<div class="footer_content">
+		<div class="footer_baseline"><§> #RulesAsCode</div>
```

Comment:
Please make sure to encode these HTML signs throughout the codebase!
```suggestion
		<div class="footer_baseline">&lt;§&gt; #RulesAsCode</div>
```